parachain/src/primitives.rs

```diff
@@ -232,7 +242,7 @@ pub struct ValidationParams {
 }
 
 /// The result of parachain validation.
-// TODO: egress and balance uploads (https://github.com/paritytech/polkadot/issues/220)
+// TODO: balance uploads (https://github.com/paritytech/polkadot/issues/220)
```

Comment:
I think this can be removed now - the functionality of "balance uploads" is handled by upward messages and balances module, right?

Reply:
Perhaps, I just don't know what is a balance upload